fmriprep/workflows/anatomical.py

```diff
@@ -572,12 +574,15 @@ def init_surface_recon_wf(omp_nthreads, hires, name='surface_recon_wf'):
         # reoriented image
         (inputnode, fs_transform, [('t1w', 'target_image')]),
         (autorecon1, fs_transform, [('T1', 'moving_image')]),
+        (inputnode, fsl2lta, [('t1w', 'target_file')]),
+        (autorecon1, fsl2lta, [('T1', 'source_file')]),
+        (fs_transform, fsl2lta, [('fsl_file', 'in_fsl')]),
```

Comment:
@oesteban These are the new lines. `lta_convert` needs to know the source and target to create a new LTA affine.

Reply:
LGTM!